internal/scaffold/olm-catalog/csv.go

```diff
@@ -263,12 +273,10 @@ func getEmptyRequiredCSVFields(csv *olmapiv1alpha1.ClusterServiceVersion) (field
 	if csv.Spec.Description == "" {
 		fields = append(fields, "spec.description")
 	}
-	if len(csv.Spec.Keywords) == 0 {
+	if len(csv.Spec.Keywords) == 0 || len(csv.Spec.Keywords[0]) == 0 {
 		fields = append(fields, "spec.keywords")
 	}
-	if len(csv.Spec.Maintainers) == 0 {
```

Comment:
Keep this around so we maintain current behavior. We can remove it in another PR.